src/spec/crypto-conditions.md

```diff
@@ -103,11 +103,26 @@ informative:
         target: http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57_part1_rev3_general.pdf
         date: 2012-07
         author:
-            - fullname: Elaine Barker
-            - fullname: William Barker
-            - fullname: William Burr
-            - fullname: William Polk
-            - fullname: Miles Smid
+            - 
```

Comment:
Without these changes, I was unable to get the xml2rfc tool to properly generate an HTML variant of the specification document.